planner/core/testdata/plan_suite_unexported_out.json

```diff
@@ -628,8 +628,7 @@
       },
       {
         "1": [
-          "test.t.a",
-          "test.t.d"
+          "test.t.a"
```

Comment:
Can you explain why this change? Rest LGTM

Reply:
What I see is sort has been ignored. So the column disappears in column pruning.